cernopendata/modules/fixtures/data/records/opera-ecc-datasets.json

```diff
@@ -223,7 +223,7 @@
         "variable": "trType"
       }
     ],
-    "date_created": "2010-2012",
+    "date_created": "2008-2012",
```

Comment:
I changed this based on the description for the methodology below

Reply:
For the tau records, since we know the concrete year for each of the 10 events, perhaps you can use the lowest and highest to denominate exact year interval...